CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [6.32.1](https://github.com/algolia/react-instantsearch/compare/v6.32.0...v6.32.1) (2022-08-26)
+
+
+### Bug Fixes
+
+* **useInstantSearch:** `results` is `null` when first search is stalled ([#3597](https://github.com/algolia/react-instantsearch/issues/3597)) ([6f71d78](https://github.com/algolia/react-instantsearch/commit/6f71d78868fde55a3f9c4460edc337a1e99df4f9))
```

Comment:
```suggestion
* **useInstantSearch:** prevent `results` from being `null` when first search is stalled ([#3597](https://github.com/algolia/react-instantsearch/issues/3597)) ([6f71d78](https://github.com/algolia/react-instantsearch/commit/6f71d78868fde55a3f9c4460edc337a1e99df4f9))
```

Reply:
Thanks ! Makes more sense to mention how it's been fixed rather than just the issue itself indeed :)